### PR TITLE
[4.0] Move searchtools, sort and item count to dropdown

### PR DESCRIPTION
**Pull request by ciar4n**
Pull Request for Issue # .

### Summary of Changes
Moves searchtools, sort and item count to a dropdown menu under a 'Table Options' toggle in list views.

### Testing Instructions
Apply patch and navigate to list view (Articles).

### Before
searchtools1

### After
searchtools2

### Documentation Changes Required
Yes

### CODE REVIEW

**Comment by brianteeman** [comment]
Can you make the dropdown collapse when you click away please

**Comment by ghost** [comment]
I have tested this item :white_check_mark: successfully on abdd678c8ff2f7a04c56865dc2463a18b985ec7f<br /><br />Hint: Dropdown-Collapse works if changing Position iE. "::None::" to "- Select Position -".<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/17059">issues.joomla.org/tracker/joomla-cms/17059</a>.</sub>

**Comment by ciar4n** [comment]
@brianteeman Dropdown now closes with click away

**Comment by ghost** [comment]
I have tested this item :white_check_mark: successfully on 1e72e40338cbb760483a4b0900eac29da523164b<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/17059">issues.joomla.org/tracker/joomla-cms/17059</a>.</sub>

File: administrator/templates/atum/scss/blocks/_searchtools.scss [hunk 1]
```diff
@@ -14,16 +15,50 @@
...
+      display: flex;
+    }
+
+  }
+
+  .js-stools-field-filter {
+    .custom-select {
+      &.active {
+        color: #fff;
+        background: #fefefe url("data:image/svg+xml;charset=utf8,%3Csvg xmlns=\'http://www.w3.org/2000/svg\' viewBox=\'0 0 4 5\'%3E%3Cpath fill=\'%23fff\' d=\'M2 0L0 2h4zm0 5L0 3h4z\'/%3E%3C/svg%3E") no-repeat scroll right .75rem center / 8px 10px;
```
Comment (C-Lodder) [inline administrator/templates/atum/scss/blocks/_searchtools.scss:42]: `#fefefe` >> `$white-offset`

File: media/system/js/searchtools.js [hunk 2]
```diff
@@ -110,7 +110,7 @@
 				self.filterButton.addEventListener('click', function(e) {
 					self.toggleFilters();
 					e.stopPropagation();
-					e.preventDefault();
+					e.preventDefault();
```
Comment (C-Lodder) [inline media/system/js/searchtools.js:113]: Remove whitespace after `e.preventDefault();`

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on 1e72e40338cbb760483a4b0900eac29da523164b<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/17059">issues.joomla.org/tracker/joomla-cms/17059</a>.</sub>

**Comment by brianteeman** [comment]
works great with core. Looks odd with com-patchtester. not sure if thats an issue or not<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/17059">issues.joomla.org/tracker/joomla-cms/17059</a>.</sub>

**Comment by mbabker** [comment]
com_patchtester doesn't use search tools since it's not using the "legacy" MVC.  So I'm kind of winging it in the 4.0 layouts to make it look similar.

**Comment by brianteeman** [comment]
@mbabker all good then ;)

**Review by C-Lodder — APPROVED** [review]

**Pull request merged**